stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/Call.kt

```diff
@@ -263,6 +270,7 @@ public class Call(
                 audioLevelOutputHelper.rampToValue(it)
             }
         }
+        observeTranscription()
```

Comment:
I need some clarity here
Should we include this function in this PR:  https://github.com/GetStream/stream-video-android/pull/1234

@aleksandar-apostolov 

**Reference: ios code:**
https://github.com/GetStream/stream-video-swift/blob/1bf0b61a13bab7f9360e452ea1cc1526ea1b3cb0/Sources/StreamVideo/Call.swift#L83

Reply:
No, the events will be propagated and the `transcribing` boolean will be updated.

Reply:
Okay